<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 3 Genomics | Jacob McPherson, PharmD</title>
  <meta name="description" content="For the betterment of man-kind" />
  <meta name="generator" content="bookdown 0.41 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 3 Genomics | Jacob McPherson, PharmD" />
  <meta property="og:type" content="book" />
  
  <meta property="og:description" content="For the betterment of man-kind" />
  <meta name="github-repo" content="jacobkmcpherson/jacobkmcpherson" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 3 Genomics | Jacob McPherson, PharmD" />
  
  <meta name="twitter:description" content="For the betterment of man-kind" />
  

<meta name="author" content="Jacob McPherson" />


<meta name="date" content="2025-01-07" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="statistics.html"/>
<link rel="next" href="pharmacology.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>



<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">A Minimal Book Example</a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> About</a>
<ul>
<li class="chapter" data-level="1.1" data-path="index.html"><a href="index.html#usage"><i class="fa fa-check"></i><b>1.1</b> Usage</a></li>
<li class="chapter" data-level="1.2" data-path="index.html"><a href="index.html#render-book"><i class="fa fa-check"></i><b>1.2</b> Render book</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="statistics.html"><a href="statistics.html"><i class="fa fa-check"></i><b>2</b> Statistics</a>
<ul>
<li class="chapter" data-level="2.1" data-path="statistics.html"><a href="statistics.html#introductory-r"><i class="fa fa-check"></i><b>2.1</b> Introductory R</a></li>
<li class="chapter" data-level="2.2" data-path="statistics.html"><a href="statistics.html#advanced-r"><i class="fa fa-check"></i><b>2.2</b> Advanced R</a></li>
<li class="chapter" data-level="2.3" data-path="statistics.html"><a href="statistics.html#introductory-statistical-programming"><i class="fa fa-check"></i><b>2.3</b> Introductory Statistical Programming</a></li>
<li class="chapter" data-level="2.4" data-path="statistics.html"><a href="statistics.html#intermediate-statistical-programming"><i class="fa fa-check"></i><b>2.4</b> Intermediate Statistical Programming</a></li>
<li class="chapter" data-level="2.5" data-path="statistics.html"><a href="statistics.html#advanced-statistical-programming"><i class="fa fa-check"></i><b>2.5</b> Advanced Statistical Programming</a></li>
<li class="chapter" data-level="2.6" data-path="statistics.html"><a href="statistics.html#data-visualization"><i class="fa fa-check"></i><b>2.6</b> Data Visualization</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="genomics.html"><a href="genomics.html"><i class="fa fa-check"></i><b>3</b> Genomics</a>
<ul>
<li class="chapter" data-level="3.1" data-path="genomics.html"><a href="genomics.html#scientific-programming-bioinformatics-computational-biology"><i class="fa fa-check"></i><b>3.1</b> **Scientific Programming: Bioinformatics &amp; Computational Biology**</a>
<ul>
<li class="chapter" data-level="3.1.1" data-path="genomics.html"><a href="genomics.html#genomics-1"><i class="fa fa-check"></i><b>3.1.1</b> Genomics</a></li>
<li class="chapter" data-level="3.1.2" data-path="genomics.html"><a href="genomics.html#metagenomics"><i class="fa fa-check"></i><b>3.1.2</b> Metagenomics</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="pharmacology.html"><a href="pharmacology.html"><i class="fa fa-check"></i><b>4</b> Pharmacology</a>
<ul>
<li class="chapter" data-level="4.1" data-path="pharmacology.html"><a href="pharmacology.html#receptor-theory"><i class="fa fa-check"></i><b>4.1</b> Receptor Theory</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="favorite-journals.html"><a href="favorite-journals.html"><i class="fa fa-check"></i><b>5</b> Favorite Journals</a></li>
<li class="chapter" data-level="6" data-path="artificial-intelligence.html"><a href="artificial-intelligence.html"><i class="fa fa-check"></i><b>6</b> Artificial Intelligence</a></li>
<li class="chapter" data-level="7" data-path="computer-science.html"><a href="computer-science.html"><i class="fa fa-check"></i><b>7</b> Computer Science</a>
<ul>
<li class="chapter" data-level="7.1" data-path="computer-science.html"><a href="computer-science.html#computer-science-communities"><i class="fa fa-check"></i><b>7.1</b> Computer science communities</a></li>
<li class="chapter" data-level="7.2" data-path="computer-science.html"><a href="computer-science.html#containerization-learning-resources"><i class="fa fa-check"></i><b>7.2</b> Containerization Learning Resources</a></li>
<li class="chapter" data-level="7.3" data-path="computer-science.html"><a href="computer-science.html#academic-programming"><i class="fa fa-check"></i><b>7.3</b> **Academic Programming**</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="structural-biology.html"><a href="structural-biology.html"><i class="fa fa-check"></i><b>8</b> Structural Biology</a>
<ul>
<li class="chapter" data-level="8.1" data-path="structural-biology.html"><a href="structural-biology.html#cryogenic-electronic-microscopy-cryoem"><i class="fa fa-check"></i><b>8.1</b> Cryogenic electronic microscopy (CryoEM)</a></li>
<li class="chapter" data-level="8.2" data-path="structural-biology.html"><a href="structural-biology.html#x-ray-crystallography"><i class="fa fa-check"></i><b>8.2</b> X-Ray Crystallography</a></li>
</ul></li>
<li class="chapter" data-level="9" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i><b>9</b> References:</a>
<ul>
<li class="chapter" data-level="9.1" data-path="references.html"><a href="references.html#genomics-2"><i class="fa fa-check"></i><b>9.1</b> Genomics</a></li>
</ul></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Jacob McPherson, PharmD</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="genomics" class="section level1 hasAnchor" number="3">
<h1><span class="header-section-number">Chapter 3</span> Genomics<a href="genomics.html#genomics" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<div id="scientific-programming-bioinformatics-computational-biology" class="section level2 hasAnchor" number="3.1">
<h2><span class="header-section-number">3.1</span> **Scientific Programming: Bioinformatics &amp; Computational Biology**<a href="genomics.html#scientific-programming-bioinformatics-computational-biology" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<div id="genomics-1" class="section level3 hasAnchor" number="3.1.1">
<h3><span class="header-section-number">3.1.1</span> Genomics<a href="genomics.html#genomics-1" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p><a href="%5Bhttps://ncbiinsights.ncbi.nlm.nih.gov/">National Center for Biotechnology and Information (NCBI)</a>](<a href="https://ncbiinsights.ncbi.nlm.nih.gov/" class="uri">https://ncbiinsights.ncbi.nlm.nih.gov/</a>))</p>
<p><a href="%5Bhttps://www.bv-brc.org/">The Bacterial and Viral Bioinformatics Resource Center (BVBRC)</a>](<a href="https://www.bv-brc.org/" class="uri">https://www.bv-brc.org/</a>))</p>
<p><a href="%5Bhttps://www.embl.org/">European Molecular Biology Laboratories (EMBL)</a>](<a href="https://www.embl.org/" class="uri">https://www.embl.org/</a>)) <a href="%5Bhttps://www.ebi.ac.uk/research">European Bioinformatics Institute (EBI)</a>](<a href="https://www.ebi.ac.uk/research" class="uri">https://www.ebi.ac.uk/research</a>))</p>
<p><a href="%5Bhttps://www.qiagen.com/us/knowledge-and-support/knowledge-hub">QIAGENâ€™s Knowledge Hub</a>,](<a href="https://www.qiagen.com/us/knowledge-and-support/knowledge-hub" class="uri">https://www.qiagen.com/us/knowledge-and-support/knowledge-hub</a>),) <a href="%5Bhttps://www.qiagen.com/us/knowledge-and-support/knowledge-hub/bench-guide">Bench Guide</a>](<a href="https://www.qiagen.com/us/knowledge-and-support/knowledge-hub/bench-guide" class="uri">https://www.qiagen.com/us/knowledge-and-support/knowledge-hub/bench-guide</a>)) and <a href="%5Bhttps://digitalinsights.qiagen.com/">Digital Insights</a>,](<a href="https://digitalinsights.qiagen.com/" class="uri">https://digitalinsights.qiagen.com/</a>),)</p>
<p><a href="%5Bhttps://www.sib.swiss/">Swiss Institute for Bioinformatics (SIB)</a>](<a href="https://www.sib.swiss/" class="uri">https://www.sib.swiss/</a>)) which <a href="%5Bhttps://github.com/GeertvanGeest">Geert van Geest</a>](<a href="https://github.com/GeertvanGeest" class="uri">https://github.com/GeertvanGeest</a>)) introduced me to the SIBâ€™s <a href="%5Bhttps://github.com/sib-swiss/AWS-docker">AWS-Docker</a>](<a href="https://github.com/sib-swiss/AWS-docker" class="uri">https://github.com/sib-swiss/AWS-docker</a>)) for getting RStudio Server, Jupyter and VSCode running on an AWS EC2 using Docker</p>
<p><a href="%5Bhttps://www.thermofisher.com/us/en/home/technical-resources/learning-centers.html">Thermo Fisherâ€™s Learning Centers</a>](<a href="https://www.thermofisher.com/us/en/home/technical-resources/learning-centers.html" class="uri">https://www.thermofisher.com/us/en/home/technical-resources/learning-centers.html</a>)) and</p>
<p><a href="%5Bhttps://www.thermofisher.com/us/en/home/digital-science/thermo-fisher-connect.html">Education Connect</a>,](<a href="https://www.thermofisher.com/us/en/home/digital-science/thermo-fisher-connect.html" class="uri">https://www.thermofisher.com/us/en/home/digital-science/thermo-fisher-connect.html</a>),)</p>
<p><a href="%5Bhttps://www.illumina.com/science/education.html">Illumina</a>,](<a href="https://www.illumina.com/science/education.html" class="uri">https://www.illumina.com/science/education.html</a>),)</p>
<p>The user manual for the <a href="%5Bhttps://resources.qiagenbioinformatics.com/manuals/clcmgm/300/index.php?manual=Create_K_mer_Tree.html">k-mer trees</a>](<a href="https://resources.qiagenbioinformatics.com/manuals/clcmgm/300/index.php?manual=Create_K_mer_Tree.html" class="uri">https://resources.qiagenbioinformatics.com/manuals/clcmgm/300/index.php?manual=Create_K_mer_Tree.html</a>)) and <a href="%5Bhttps://resources.qiagenbioinformatics.com/manuals/clcmgm/300/index.php?manual=Create_SNP_Tree.html">SNP trees</a>](<a href="https://resources.qiagenbioinformatics.com/manuals/clcmgm/300/index.php?manual=Create_SNP_Tree.html" class="uri">https://resources.qiagenbioinformatics.com/manuals/clcmgm/300/index.php?manual=Create_SNP_Tree.html</a>)) are relatively more straight forward **WHEN USING WORKFLOWS**. Nonetheless, their visualization could use improvement; I naturally turn to **python** for bioinformatics and **R** for visualization</p>
</div>
<div id="metagenomics" class="section level3 hasAnchor" number="3.1.2">
<h3><span class="header-section-number">3.1.2</span> Metagenomics<a href="genomics.html#metagenomics" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p><a href="%5Bhttps://www.illumina.com/areas-of-interest/microbiology/microbial-sequencing-methods/16s-rrna-sequencing.html">16s rRNA gene sequencing with Illumina</a>,](<a href="https://www.illumina.com/areas-of-interest/microbiology/microbial-sequencing-methods/16s-rrna-sequencing.html" class="uri">https://www.illumina.com/areas-of-interest/microbiology/microbial-sequencing-methods/16s-rrna-sequencing.html</a>),) which feeds into either the current gold-standard open-source (python) tool <a href="%5Bhttps://qiime2.org/">QIIME2</a>](<a href="https://qiime2.org/" class="uri">https://qiime2.org/</a>)) by <a href="%5Bhttps://www.nature.com/articles/s41587-019-0209-9">Bolyen *et al.* 2019</a>,](<a href="https://www.nature.com/articles/s41587-019-0209-9" class="uri">https://www.nature.com/articles/s41587-019-0209-9</a>),) the superseded (C++) gold-standard tool, <a href="%5Bhttps://github.com/mothur/mothur">Mothur</a>](<a href="https://github.com/mothur/mothur" class="uri">https://github.com/mothur/mothur</a>)) by <a href="%5Bhttps://journals.asm.org/doi/10.1128/AEM.01541-09">Schloss *et al.* 2009</a>,](<a href="https://journals.asm.org/doi/10.1128/AEM.01541-09" class="uri">https://journals.asm.org/doi/10.1128/AEM.01541-09</a>),) which has a <a href="%5Bhttps://training.galaxyproject.org/archive/2021-10-01/topics/metagenomics/tutorials/mothur-miseq-sop/tutorial.html">16S rRNA gene sequencing tutorial</a>](<a href="https://training.galaxyproject.org/archive/2021-10-01/topics/metagenomics/tutorials/mothur-miseq-sop/tutorial.html" class="uri">https://training.galaxyproject.org/archive/2021-10-01/topics/metagenomics/tutorials/mothur-miseq-sop/tutorial.html</a>))</p>
<p><a href="%5Bhttps://resources.qiagenbioinformatics.com/manuals/clcmgm/300/index.php?manual=Introduction_Metagenomics.html">16s rRNA gene sequencing with CLC</a>](<a href="https://resources.qiagenbioinformatics.com/manuals/clcmgm/300/index.php?manual=Introduction_Metagenomics.html" class="uri">https://resources.qiagenbioinformatics.com/manuals/clcmgm/300/index.php?manual=Introduction_Metagenomics.html</a>)) with <a href="%5Bhttps://digitalinsights.qiagen.com/wp-content/uploads/2016/05/Characterizing-the-Microbiome-through-Targeted-Sequencing-of-Bacterial-16S-rRNA-and-Fungal-ITS-Regions_White-Paper_QIAGEN-Bioinformatics_0518_ww.pdf">associated white paper</a>:](<a href="https://digitalinsights.qiagen.com/wp-content/uploads/2016/05/Characterizing-the-Microbiome-through-Targeted-Sequencing-of-Bacterial-16S-rRNA-and-Fungal-ITS-Regions_White-Paper_QIAGEN-Bioinformatics_0518_ww.pdf" class="uri">https://digitalinsights.qiagen.com/wp-content/uploads/2016/05/Characterizing-the-Microbiome-through-Targeted-Sequencing-of-Bacterial-16S-rRNA-and-Fungal-ITS-Regions_White-Paper_QIAGEN-Bioinformatics_0518_ww.pdf</a>):) The CLC workflow for 16S follows an <a href="%5Bhttps://resources.qiagenbioinformatics.com/manuals/clcmgm/300/index.php?manual=Amplicon_based_OTU_clustering.html">amplicon-based OTU clustering workflow</a>](<a href="https://resources.qiagenbioinformatics.com/manuals/clcmgm/300/index.php?manual=Amplicon_based_OTU_clustering.html" class="uri">https://resources.qiagenbioinformatics.com/manuals/clcmgm/300/index.php?manual=Amplicon_based_OTU_clustering.html</a>)) that uses <a href="">read trimming</a> using their <a href="%5Bhttps://resources.qiagenbioinformatics.com/manuals/clcassemblycell/400/index.php?manual=Quality_trimming.html">â€˜clc_quality_trimâ€™ program</a>,](<a href="https://resources.qiagenbioinformatics.com/manuals/clcassemblycell/400/index.php?manual=Quality_trimming.html" class="uri">https://resources.qiagenbioinformatics.com/manuals/clcassemblycell/400/index.php?manual=Quality_trimming.html</a>),) but **I would rather use** <a href="%5Bhttp://www.usadellab.org/cms/?page=trimmomatic">trimmomatic</a>](<a href="http://www.usadellab.org/cms/?page=trimmomatic" class="uri">http://www.usadellab.org/cms/?page=trimmomatic</a>)) by <a href="%5Bhttps://pubmed.ncbi.nlm.nih.gov/24695404/">Bolger, Lihse &amp; Usadel, 2014</a>;](<a href="https://pubmed.ncbi.nlm.nih.gov/24695404/" class="uri">https://pubmed.ncbi.nlm.nih.gov/24695404/</a>);) <a href="%5Bhttps://resources.qiagenbioinformatics.com/manuals/clcmgm/300/index.php?manual=Filter_Samples_Based_on_Number_Reads.html">filtering samples based on the number of reads</a>;](<a href="https://resources.qiagenbioinformatics.com/manuals/clcmgm/300/index.php?manual=Filter_Samples_Based_on_Number_Reads.html" class="uri">https://resources.qiagenbioinformatics.com/manuals/clcmgm/300/index.php?manual=Filter_Samples_Based_on_Number_Reads.html</a>);) *de novo* or reference-based [OTU clustering] <a href="https://resources.qiagenbioinformatics.com/manuals/clcmgm/300/index.php?manual=OTU_clustering_parameters.html">https://resources.qiagenbioinformatics.com/manuals/clcmgm/300/index.php?manual=OTU_clustering_parameters.html);</a>;) <a href="%5Bhttps://resources.qiagenbioinformatics.com/manuals/clcmgm/300/index.php?manual=Remove_OTUs_with_Low_Abundance.html">removal of low abundance OTUs</a>;](<a href="https://resources.qiagenbioinformatics.com/manuals/clcmgm/300/index.php?manual=Remove_OTUs_with_Low_Abundance.html" class="uri">https://resources.qiagenbioinformatics.com/manuals/clcmgm/300/index.php?manual=Remove_OTUs_with_Low_Abundance.html</a>);) <a href="%5Bhttps://resources.qiagenbioinformatics.com/manuals/clcmgm/300/index.php?manual=Abundance_analysis.html">OTU abundance analysis</a>](<a href="https://resources.qiagenbioinformatics.com/manuals/clcmgm/300/index.php?manual=Abundance_analysis.html" class="uri">https://resources.qiagenbioinformatics.com/manuals/clcmgm/300/index.php?manual=Abundance_analysis.html</a>)) **but I prefer R for this;**</p>
<p><a href="%5Bhttps://resources.qiagenbioinformatics.com/manuals/clcmgm/300/index.php?manual=Align_OTUs_with_MUSCLE.html">OTU nucleotide alignment with MUSCLE</a>](<a href="https://resources.qiagenbioinformatics.com/manuals/clcmgm/300/index.php?manual=Align_OTUs_with_MUSCLE.html" class="uri">https://resources.qiagenbioinformatics.com/manuals/clcmgm/300/index.php?manual=Align_OTUs_with_MUSCLE.html</a>)) by <a href="%5Bhttps://academic.oup.com/nar/article/32/5/1792/2380623?login=true">Edgar, 2004</a>](<a href="https://academic.oup.com/nar/article/32/5/1792/2380623?login=true" class="uri">https://academic.oup.com/nar/article/32/5/1792/2380623?login=true</a>)) to <a href="%5Bhttp://resources.qiagenbioinformatics.com/manuals/clcgenomicsworkbench/current/index.php?manual">generate a maximum likelihood phylogenetic tree</a>,](<a href="http://resources.qiagenbioinformatics.com/manuals/clcgenomicsworkbench/current/index.php?manual" class="uri">http://resources.qiagenbioinformatics.com/manuals/clcgenomicsworkbench/current/index.php?manual</a>),) input for the <a href="%5Bhttps://resources.qiagenbioinformatics.com/manuals/clcmgm/300/index.php?manual=Estimate_Alpha_Beta_Diversities_workflow.html">alpha- and beta-diversity workflow</a>](<a href="https://resources.qiagenbioinformatics.com/manuals/clcmgm/300/index.php?manual=Estimate_Alpha_Beta_Diversities_workflow.html" class="uri">https://resources.qiagenbioinformatics.com/manuals/clcmgm/300/index.php?manual=Estimate_Alpha_Beta_Diversities_workflow.html</a>)) **but I prefer <a href="%5Bhttps://vegandevs.github.io/vegan/index.html">vegan</a>](<a href="https://vegandevs.github.io/vegan/index.html" class="uri">https://vegandevs.github.io/vegan/index.html</a>)) for this**</p>
<p>The microbial , <a href="%5Bhttps://github.com/picrust/picrust2">PICRUST2</a>,](<a href="https://github.com/picrust/picrust2" class="uri">https://github.com/picrust/picrust2</a>),) and the <a href="%5Bhttps://portal.hmpdacc.org/">interactive Human Microbiome Project (iHMP)</a>](<a href="https://portal.hmpdacc.org/" class="uri">https://portal.hmpdacc.org/</a>))</p>
<p><a href="%5Bhttps://www.illumina.com/areas-of-interest/microbiology/microbial-sequencing-methods/shotgun-metagenomic-sequencing.html">Illumina SGS</a>](<a href="https://www.illumina.com/areas-of-interest/microbiology/microbial-sequencing-methods/shotgun-metagenomic-sequencing.html" class="uri">https://www.illumina.com/areas-of-interest/microbiology/microbial-sequencing-methods/shotgun-metagenomic-sequencing.html</a>))</p>
<p>In CLC, whole metagenome shotgun sequencing <a href="%5Bhttps://resources.qiagenbioinformatics.com/manuals/clcmgm/300/index.php?manual=Functional_analysis.html">functional analysis</a>](<a href="https://resources.qiagenbioinformatics.com/manuals/clcmgm/300/index.php?manual=Functional_analysis.html" class="uri">https://resources.qiagenbioinformatics.com/manuals/clcmgm/300/index.php?manual=Functional_analysis.html</a>)) first includes the user <a href="%5Bhttps://resources.qiagenbioinformatics.com/manuals/clcmgm/300/index.php?manual=De_Novo_Assemble_Metagenome.html#sec:de_novo_assemble_metagenome">*de novo* assembling a metagenome</a>,](<a href="https://resources.qiagenbioinformatics.com/manuals/clcmgm/300/index.php?manual=De_Novo_Assemble_Metagenome.html#sec:de_novo_assemble_metagenome" class="uri">https://resources.qiagenbioinformatics.com/manuals/clcmgm/300/index.php?manual=De_Novo_Assemble_Metagenome.html#sec:de_novo_assemble_metagenome</a>),) followed by annotation of the coding sequence (CDS) track with</p>
<p><a href="%5Bhttps://resources.qiagenbioinformatics.com/manuals/clcmgm/300/index.php?manual=Annotate_CDS_with_Best_BLAST_Hit.html#sec:annotate_cds_with_blast">BLAST</a>,](<a href="https://resources.qiagenbioinformatics.com/manuals/clcmgm/300/index.php?manual=Annotate_CDS_with_Best_BLAST_Hit.html#sec:annotate_cds_with_blast" class="uri">https://resources.qiagenbioinformatics.com/manuals/clcmgm/300/index.php?manual=Annotate_CDS_with_Best_BLAST_Hit.html#sec:annotate_cds_with_blast</a>),) <a href="%5Bhttps://resources.qiagenbioinformatics.com/manuals/clcmgm/300/index.php?manual=Annotate_CDS_with_Pfam_Domains.html#sec:annotate_cds_with_pfam">Pfam domains</a>,](<a href="https://resources.qiagenbioinformatics.com/manuals/clcmgm/300/index.php?manual=Annotate_CDS_with_Pfam_Domains.html#sec:annotate_cds_with_pfam" class="uri">https://resources.qiagenbioinformatics.com/manuals/clcmgm/300/index.php?manual=Annotate_CDS_with_Pfam_Domains.html#sec:annotate_cds_with_pfam</a>),) and/or</p>
<p><a href="%5Bhttps://resources.qiagenbioinformatics.com/manuals/clcmgm/300/index.php?manual=Download_GO_Database.html#sec:download_go">Gene Ontology (GO)</a>.](<a href="https://resources.qiagenbioinformatics.com/manuals/clcmgm/300/index.php?manual=Download_GO_Database.html#sec:download_go" class="uri">https://resources.qiagenbioinformatics.com/manuals/clcmgm/300/index.php?manual=Download_GO_Database.html#sec:download_go</a>).) Then you map the original reads back to the annotated contigs using the â€˜Map Reads to Referenceâ€™ in the <a href="%5Bhttps://resources.qiagenbioinformatics.com/manuals/clcmgm/300/index.php?manual=Build_Functional_Profile.html#sec:functional_profile">Build Functional Profile</a>](<a href="https://resources.qiagenbioinformatics.com/manuals/clcmgm/300/index.php?manual=Build_Functional_Profile.html#sec:functional_profile" class="uri">https://resources.qiagenbioinformatics.com/manuals/clcmgm/300/index.php?manual=Build_Functional_Profile.html#sec:functional_profile</a>)) tool. The resulting output can be visualized using stacked bar charts and sunburst plots in <a href="%5Bhttps://resources.qiagenbioinformatics.com/manuals/clcmgm/300/index.php?manual=Visualization_OTU_abundance_tables.html#sec:visualizationotu">Visualization of the OTU abundance table</a>](<a href="https://resources.qiagenbioinformatics.com/manuals/clcmgm/300/index.php?manual=Visualization_OTU_abundance_tables.html#sec:visualizationotu" class="uri">https://resources.qiagenbioinformatics.com/manuals/clcmgm/300/index.php?manual=Visualization_OTU_abundance_tables.html#sec:visualizationotu</a>)) **but as you guessed, I prefer R for this.** As you might expect, **I might use** the open-source Linux OS (python) tool <a href="%5Bhttps://github.com/picrust/picrust2">PICRUSt2</a>](<a href="https://github.com/picrust/picrust2" class="uri">https://github.com/picrust/picrust2</a>)) by <a href="%5Bhttps://www.nature.com/articles/s41587-020-0548-6">Douglas *et al.* 2020</a>](<a href="https://www.nature.com/articles/s41587-020-0548-6" class="uri">https://www.nature.com/articles/s41587-020-0548-6</a>)) to do this too.</p>

</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="statistics.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="pharmacology.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/jkmcpherson/jkmcpherson/edit/master/02-Genomics.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["_main.pdf", "_main.epub"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.9/latest.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
